test/runtests.py

```diff
@@ -89,7 +89,11 @@ def test_path(folder, is_file):
                 print "Javascript file doesn't exist (" + js_file + ")"
                 sys.exit(1)
 
-            p = SP.Popen([ch_path, js_file], stdout=SP.PIPE, stderr=SP.STDOUT, close_fds=True)
+            working_path = os.path.dirname(js_file)
+            file_name = os.path.basename(js_file)
+            p = SP.Popen([ch_path, file_name], stdout=SP.PIPE, stderr=SP.STDOUT, 
+                                                             cwd=working_path,
+                                                             close_fds=True)
```

Comment:
Question about `close_fds=True`: I deleted it in my PR, because python complains this parameter combination is invalid:

```
If close_fds is true, all file descriptors except 0, 1 and 2 will be closed before the child process is executed. (Unix only). Or, on Windows, if close_fds is true then no handles will be inherited by the child process. Note that on Windows, you cannot set close_fds to true and also redirect the standard handles by setting stdin, stdout or stderr.
```

What were your purpose to use this parameter? To close the new pipe? Do we need to worry about closing the new pipe?